skunkworks_crow/src/main/java/org/odk/share/fragments/BlankFormsFragment.java

```diff
@@ -193,8 +193,12 @@ private void toggleButtonLabel() {
         }
         if (selectedForms.isEmpty()) {
             sendButton.setText(getString(R.string.send_forms));
+            sendButton.setBackground(getResources().getDrawable(R.drawable.selector_bottom_recive));
```

Comment:
```java
            sendButton.setText(getString(R.string.send_forms));
            sendButton.setBackground(getResources().getDrawable(R.drawable.selector_bottom_recive));
            sendButton.setTextColor(getResources().getColor(R.color.colorPrimary));
```
It will be good for you to extract those similar code to a function like `updateSendButton(String newText, @DrawableRes int newDrawableRes, @ColorRes int newTextColor)`.